docs/tutorials/civo.md

```diff
@@ -0,0 +1,187 @@
+# Setting up ExternalDNS for Services on Civo
+
+This tutorial describes how to setup ExternalDNS for usage within a Kubernetes cluster using Civo DNS Manager.
+
+Make sure to use **>=1.7.1** version of ExternalDNS for this tutorial.
```

Comment:
The latest ExternalDNS version is v0.12.2, so maybe indicate `>0.12.2` it should be fine.